src/debug/inc/dbgipcevents.h

```diff
@@ -1172,9 +1172,9 @@ struct MSLAYOUT DebuggerREGDISPLAY
     #define DebuggerIPCE_FloatCount 32
 
     SIZE_T  X[29];
-    SIZE_T  SP;
     SIZE_T  FP;
     SIZE_T  LR;
+    SIZE_T  SP;
```

Comment:
On `arm64` 
- `FP` is register `X29`, 
- `LR` is register `X30`, 
- `SP` is register `X31`, 

Reorder this structure to make it consistent with all other arm64 context structures